extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -359,6 +365,7 @@ export class AmpStoryPage extends AMP.BaseElement {
     if (this.isActive()) {
       this.advancement_.start();
       this.maybeStartAnimations();
+      this.findAndPrepareEmbeddedComponents();
```

Comment:
Sounds good, but we also need to do it on `resumeCallback` in case we land on the page directly.

You'll notice I added two different timeouts: one for when we call this method from the preloading and a different one in the `resumeCallback`.  The reasoning is that we can allow a longer timeout on a page that isn't active yet.